src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,36 @@
 import React from 'react';
 import './MoviesList.scss';
+import { MovieCard } from '../MovieCard';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+interface Movie {
```

Comment:
You should transfer this interface in seperate file and import it in this file. Maybe now it isn't be a mistake, because in task we have only one interfase, but I think it'll be better to create seperate file with types.